docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM store/oracle/serverjre:8
```

Comment:
Since this file is in the "docker" directory (this is awesome btw)
 can you please make a change to src/test/TestRequiredFilesExist.java to fix the file path for the DockerFile? 

line 10 should look like this:
```
    @Test public void checkDockerExists() {
        boolean dockerExists = new File("./docker/Dockerfile").exists() ||
        new File("./docker/Docker").exists();
        assertTrue(dockerExists);
    }
```